src/main/java/org/spongepowered/api/resource/pack/PackTypes.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.resource.pack;
+
+import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
+
+public class PackTypes {
+
+    // SORTFIELDS:ON
+
+    public static final PackType ASSETS = DummyObjectProvider.createFor(PackType.class, "ASSETS");
```

Comment:
Nah, this is actually resource.
![image](https://user-images.githubusercontent.com/7806504/57576981-da7eda00-7421-11e9-8cf3-8dd96e59c0b5.png)

Reply:
Yeah, but I don't really like that name.

Reply:
`RESOURCE` is already an established name. (I don't ask you to name this one with the client prefix) This name is not mapped, but rather inferred from the enum constructor parameters passed.

Well, minecraft wiki documents resource and data packs, not assets and data packs. Beware of that.